equinix/resource_metal_spot_market_request.go

```diff
@@ -135,8 +135,15 @@ func resourceMetalSpotMarketRequest() *schema.Resource {
 							Required: true,
 						},
 						"termintation_time": {
-							Type:     schema.TypeString,
-							Computed: true,
+							Type:          schema.TypeString,
+							Optional:      true,
```

Comment:
I think these should both still be `Computed: true`

Reply:
The termination time for a spot market request resource is only read from the API; we don't have any code to include it in the spot market request that is sent to the API.